crates/subspace-networking/src/protocols/subspace_connection_limits.rs

```diff
@@ -26,11 +28,24 @@ impl Behaviour {
     }
 
     /// Add to allow list some attempts of incoming connections from specified peer ID that will bypass global limits
-    pub(crate) fn add_to_incoming_allow_list(&mut self, peer: PeerId, attempts: usize) {
-        self.incoming_allow_list
-            .entry(peer)
-            .and_modify(|entry| *entry = entry.saturating_add(attempts))
-            .or_insert(attempts);
+    pub(crate) fn add_to_incoming_allow_list<IpAddresses>(
```

Comment:
I don't see corresponding changes for the outgoing connections. I recommend deleting the dead code though and recreating when you have time to push changes upstream.